openreview/profile/process/dblp_author_coreference_pre_process.js

```diff
@@ -0,0 +1,30 @@
+async function process(client, edit, invitation) {
+  client.throwErrors = true;
+  
+  const value = edit.note.content.authorids?.value;
+
+  if (!value?.replace) {
+    return Promise.reject(new OpenReviewError({ name: 'Error', message: `Only author replacement is allowed` }))
+  }
+  
+  const index = value.replace.index;
+  const authorId = value.replace.value;
```

Comment:
i think this process function need to allow null replace value so that imported papers can be unlinked

Reply:
Trying using an empty string instead of null, I added a test to verify that.

Reply:
now the author conreference get author index and author id from edit content i think it's not possible to unlink a publication